lib/Target/LLVMIR/LLVMDIScope.cpp

```diff
@@ -107,7 +107,7 @@ struct LLVMDIScopePass : public LLVMDIScopeBase<LLVMDIScopePass> {
         llvm::sys::path::parent_path(calleeFileName));
     auto lexicalBlockFileAttr = LLVM::DILexicalBlockFileAttr::get(
         context, scopeAttr, calleeFileAttr, /*discriminator=*/0);
-    Location loc = op->getLoc();
+    Location loc = calleeLoc;
```

Comment:
is this related to the LLVM upgrade?

Reply:
Yes, it's specifically for this commit: https://github.com/llvm/llvm-project/pull/74546 they changed the format a bit (see https://github.com/llvm/llvm-project/pull/74546#discussion_r1447922845)

Reply:
Interesting, thanks for the info!